### ISSUE TITLE
Support for re-exported types

### ISSUE THREAD

**Issue by tdamir** [MeirionHughes/aurelia-template-lint#112]
item.ts
```
export class Item {
  prop: string;
}
```

common.ts
```
export * from './item'
```

page.ts
```
import {Item} from "./common";
export class PageViewModel {
    items: Item[];
}
```

page.html
```
<template>
   <div repeat.for="item in items">
    ${item.propN}
   </div>
</template>
```
It correctly reports error if we import the type from item.ts. It should report an error if we import the type from common.ts too.

**Comment by MeirionHughes** [comment]
published to 0.9.11
